core/drivers/firewall/stm32_rifsc.c

```diff
@@ -28,10 +28,12 @@
 #define _RIFSC_RISC_PER0_SEMCR		U(0x104)
 #define _RIFSC_RIMC_CR			U(0xC00)
 #define _RIFSC_RIMC_ATTR0		U(0xC10)
+#if defined(CFG_STM32MP25)
 #define _RIFSC_RISAL_CFGR0_A(y)		(U(0x900) + (0x10 * ((y) - 1)))
 #define _RIFSC_RISAL_CFGR0_B(y)		(U(0x908) + (0x10 * ((y) - 1)))
 #define _RIFSC_RISAL_ADDR_A		U(0x924)
 #define _RIFSC_RISAL_ADDR_B		U(0x92C)
+#endif /* defined(CFG_STM32MP25) */
```

Comment:
```suggestion
#endif /* CFG_STM32MP25 */
```
Several occurrences.